config/prod.env.js

```diff
@@ -1,3 +1,8 @@
 module.exports = {
   NODE_ENV: '"production"'
+
```

Comment:
we should also add the stripe key in here. Thats actually a perfect candidate since its never anything other then dev or testing